src/BackgroundCells.js

```diff
@@ -62,8 +64,10 @@ class BackgroundCells extends React.Component {
       cellWrapperComponent: Wrapper,
       dayPropGetter,
       date: currentDate,
+      nowAccessor,
     } = this.props
     let { selecting, startIdx, endIdx } = this.state
+    let current = get(nowAccessor)
```

Comment:
the use of `get` is a bit much for this, since it's a required function, `nowAccesser()` is fine